app/models/data_file.rb

```diff
@@ -180,6 +182,15 @@ def related_website_list
     self.related_websites.collect(&:url).join("|")
   end
 
+  def validate_related_websites
+    related_websites = related_website_list.split("|")
```

Comment:
The existing code split related websites on '|' so I follow that. Should I
refactor code for related websites?

On 4 September 2015 at 15:15, Daniel Yazbek <notifications@github.com>
wrote:

> In app/models/data_file.rb
> <https://github.com/IntersectAustralia/dc21/pull/14#discussion_r38722210>:
>
> > @@ -180,6 +182,15 @@ def related_website_list
> >      self.related_websites.collect(&:url).join("|")
> >    end
> >
> > +  def validate_related_websites
> > +    related_websites = related_website_list.split("|")
>
> Shouldn't you split on space?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/IntersectAustralia/dc21/pull/14/files#r38722210>.
>